### ISSUE TITLE
Readme instructions error

### ISSUE THREAD

**Issue by danballance** [core-api/python-openapi-codec#23]
Hi,

Great project! Finally had a chance to check this out with a real use case of my own.

I had an issue getting the instructions to work. Not sure if it was my end or not, but what worked for me was:

```
decoders = [OpenAPICodec(), JSONCodec()]
client = Client(decoders=decoders)  
schema = client.get('http://petstore.swagger.io/v2/swagger.json', force_codec='openapi')
```

The error I get using format='openapi' is:

`coreapi.exceptions.NoCodecAvailable: Unsupported media in Content-Type header 'application/json'`

I noticed that everything seemed to work with the [cli client](https://github.com/core-api/coreapi-cli). That's when I noticed that it uses force_codec and not format.

I don't know your code base so I'm not 100% if using force_code is correct in this context, but I thought I should create a ticket in case it hits anyone else.

**Comment by lovelydinosaur** [comment]
Good catch, yes. The old style was `force_codec=<bool>` the new style should be `format=<string>`. Will look into asserting that the correct type is passed.
